inductiva/__init__.py

```diff
@@ -68,9 +68,10 @@ def _check_for_available_package_update():
 
 def _check_key():
     global _checked_key
-    if not _checked_key and os.getenv("GITHUB_ACTIONS",
-                                      "false").lower() != "true":
-        validate_api_key(api_key)
+
+    if not _checked_key and utils.format_utils.getenv_bool(
+            "GITHUB_ACTIONS", False) is not True:
```

Comment:
So it is False?

Reply:
If running on Github we will get True, and we won't enter the if statement. If we run on our pc we will get False and we will enter the if statement. 
Github action was failing because we had no api key, so we had to disable this check for GitHub actions.

Reply:
I just wanted to say the `getenv_bool` gets you a boolean, so you don't need to compare with `is not True`, you just need to negate it. My logic classes are also tingling me to use morgan's law and apply a general not like:
`not (_checked_key or utils.format_utils.getenv_bool(...)`